engine/src/test/java/io/camunda/zeebe/engine/processing/streamprocessor/StreamProcessorReplayModeTest.java

```diff
@@ -308,9 +308,6 @@ public void shouldNotSetLastProcessedPositionIfLessThanSnapshotPosition() {
             RECORD,
             writer -> writer.sourceRecordPosition(commandPositionBeforeSnapshot));
 
-    verify(replayContinuously.getMockStreamProcessorListener(), TIMEOUT)
```

Comment:
:x: This might be an issue since we no longer await that the replay was done, which might cause flakiness. I think the test here might be still green, but it is worth rewriting it with the new Extension https://github.com/camunda/zeebe/pull/10036

If you don't want to do it right now, maybe you can just uncomment the removed code, some we know what it checked before.

Reply:
Added an await and ran the test 250+ times

Reply:
thanks peter :+1: